docs/examples/arrow_example.ipynb

```diff
@@ -92,6 +118,73 @@
     "print(\"asym_load:\", asym_load_input_dtype)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The primitive types of each attribute in the arrow tables need to match to make the operation efficient.\n",
+    "A zero copy is not guaranteed if the data types from power_grid_meta_data / initialize_array are not used.\n",
+    "Note that the asymmetric type of attribute in power-grid-model has a shape of `(3,)` along with a specific type. These represent the 3 phases of electrical system.\n",
+    "Hence asymmetric attributes need to be handled specially. \n",
+    "\n",
+    "In this tutorial we use the respective primitive types for the symmetrical attributes and a `FixedSizeListArray` of the primitive types with length 3 for asymmetrical attributes. This results in them being stored as contigious memory which would enable zero copy conversion. There might be other ways to approach this problem too."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Creating a Schema\n",
+    "\n",
+    "We can make the task of assigning types easier by creating a schema based on the `DatasetType` and `ComponentType` directly from `power_grid_meta_data`. \n",
+    "They can then directly be used to construct the `RecordBatch`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-------node combined asym scehma-------\n",
+      "id: int32\n",
+      "u_rated: double\n",
+      "-------asym load combined asym scehma-------\n",
+      "id: int32\n",
+      "node: int32\n",
+      "status: int8\n",
+      "type: int8\n",
+      "p_specified: fixed_size_list<item: double>[3]\n",
+      "  child 0, item: double\n",
+      "q_specified: fixed_size_list<item: double>[3]\n",
+      "  child 0, item: double\n"
+     ]
+    }
+   ],
+   "source": [
+    "def pgm_schema(dataset_type: DatasetType, component_type: ComponentType, attributes: Iterable[str] | None = None):\n",
+    "    schemas = []\n",
+    "    component_dtype = power_grid_meta_data[dataset_type][component_type].dtype\n",
+    "    for meta_attribute, (dtype, _) in component_dtype.fields.items():\n",
+    "        if attributes is not None and meta_attribute not in attributes:\n",
+    "            continue\n",
+    "        if dtype.shape == (3,):\n",
+    "            pa_dtype = pa.list_(pa.from_numpy_dtype(dtype.base), 3)\n",
+    "        else:\n",
+    "            pa_dtype = pa.from_numpy_dtype(dtype)\n",
+    "        schemas.append((meta_attribute, pa_dtype))\n",
+    "    return pa.schema(schemas)\n",
+    "\n",
+    "\n",
+    "print(\"-------node combined asym scehma-------\")\n",
```

Comment:
phrasing is a bit weird. maybe something like `node pyarrow data schema`?

Reply:
Sure is, forgot to modify it.